recipes/emsdk_installer/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, tools
+import os
+
+
+class EmSDKInstallerConan(ConanFile):
+    name = "emsdk_installer"
+    description = "Emscripten is an Open Source LLVM to JavaScript compiler"
+    url = "https://github.com/bincrafters/conan-emsdk_installer"
+    homepage = "https://github.com/kripken/emscripten"
+    topics = ("conan", "emsdk", "emscripten", "installer", "sdk")
+    license = "MIT"
+
+    short_paths = True
+    _source_subfolder = "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = "emsdk-%s" % self.version
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def _run(self, command):
+        self.output.info(command)
+        self.run(command)
+
+    @staticmethod
+    def _create_dummy_file(directory):
+        if not os.path.isdir(directory):
+            os.makedirs(directory)
+        with open(os.path.join(directory, "dummy"), "w") as f:
+            f.write("\n")
```

Comment:
I think `tools.save` automatically creates the directory structure.
As this method can be replaced with one function call, this method can be removed all together.
```suggestion
        tools.save(os.path.join(directory, "dummy"), "\n")
```